modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/asset/model/LayoutAssetRendererFactory.java

```diff
@@ -99,6 +99,11 @@ public String getClassName() {
 		return Layout.class.getName();
 	}
 
+	@Override
+	public String getIconCssClass() {
+		return "page";
```

Comment:
@lipusz @thektan just curious if this should be "layout" instead of "page"?

Reply:
seems like a lot of the other AssetRendererFactories have their IconCssClass stay pretty consistent with TYPE, and in this class TYPE = "layout"

Reply:
@BryanEngler The "page" icon seems appropriate for this type of asset. They correspond to the clay icons documented here https://clayui.com/docs/components/icon.html

Reply:
👍🏻 ya i had no idea what this actually does